### ISSUE TITLE
extension host terminated unexpectedly vscode

### ISSUE THREAD

**Issue by Dorriss** [georgewfraser/java-language-server#106]
Issue Type: <b>Bug</b>

extension host terminated unexpectedly vscode

Extension version: 0.2.20
VS Code version: Code 1.36.1 (2213894ea0415ee8c85c5eea0d0ff81ecc191529, 2019-07-08T22:59:35.033Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz (4 x 2808)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.41GB (9.55GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by georgewfraser** [comment]
Is this still happening with the latest version?

**Comment by ash1247** [comment]
Yes for me, After I open a .java file and the moment I start to write code the extension terminates.

**Comment by ccmgDeveloper** [comment]
Sadly, I had to disable to compile my code
